clang/docs/SanitizerSpecialCaseList.rst

```diff
@@ -67,9 +67,11 @@ types specified within an ignorelist.
     int a = 2147483647; // INT_MAX
     ++a;                // Normally, an overflow with -fsanitize=signed-integer-overflow
   }
+
```

Comment:
Please do unrelated fixes in separate PRs.

Reply:
The wraps docs reference the SSCL docs and I fixed some whitespace in the example to make it more readable.

Reply:
Yes, I'ts about spaces. Even if it's useful change  just create a new PR. Even from web UI. If you had commit access, no review is needed.

Reply:
I don't have commit access, but I'll revert that whitespace change 👍